config/config.yml.example

```diff
@@ -15,13 +15,22 @@ products: # list of exact product names as they appear on Bugzilla
     components: # if you want to filter the given product by components
     - example_component_1
     - example_component_2
-organizations: # here you add the GitHub repos you care about
-  - name: opensuse
-    repositories:
-      - nailed # e.g. for github.com/opensuse/nailed
-      - other_repo
-  - name: other_organization
-    repositories:
-      - repo1
-      - repo2
+github: # here you add the Github repos you care about
+  organizations:
+    - name: opensuse
+      repositories:
+        - nailed # e.g. for github.com/opensuse/nailed
+        - other_repo
+    - name: other_organization
+      repositories:
+        - repo1
+        - repo2
+...
```

Comment:
`...` signifies the end of a YAML file, you have it now twice in the example file.